inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java

```diff
@@ -479,7 +482,7 @@ public TestContext()
                 FileUtils.deleteDirectory(new File(temporaryFolderPath));
             }
             catch (IOException e) {
-                e.printStackTrace();
+                throw new RuntimeException(e);
```

Comment:
Either a failed deletion should be ignored and logged (as before) or it would be sufficient to just let it thrown the IOException. Throwing the exception is probably the better choice.